command/daemon/command.go

```diff
@@ -8,11 +8,17 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/giantswarm/microkit/command/daemon/flag"
```

Comment:
Each package that defines flags should provide a `flag` package containing the flag structure.